apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/tally-template-a/tally-template-a.component.html

```diff
@@ -0,0 +1,938 @@
+                                                    </strong>
+                                                </h1>
+                                            </ng-container>
+                                        </td>
+                                        <td class="border-none text-right" width="33%">
+                                            <!-- This will show typeOfInvoice (Original	for	Recipient) in PDF -->
+                                        </td>
+                                    </tr>
+                                    <tr *ngIf="fieldsAndVisibility.footer.displayExportMessage?.display">
+                                        <td
+                                            colspan="100%"
+                                            class="border-none text-center"
+                                            [ngStyle]="{
+                                                'font-size': inputTemplate.fontMedium + 'px',
+                                                'white-space': 'pre-line'
+                                            }"
+                                        >
+                                            {{
+                                                activeCompany?.withPay
+                                                    ? "SUPPLY MEANT FOR EXPORT UNDER BOND OR LETTER OF UNDERTAKING WITH PAYMENT OF INTEGRATED TAX"
+                                                    : "SUPPLY MEANT FOR EXPORT UNDER BOND OR LETTER OF UNDERTAKING WITHOUT PAYMENT OF INTEGRATED TAX"
+                                            }}
+                                        </td>
+                                    </tr>
+                                </table>
+
+                                <table
+                                    *ngIf="
+                                        showGstComposition &&
+                                        (fieldsAndVisibility?.header?.showQrCode?.display ||
+                                            fieldsAndVisibility?.header?.showEInvoiceDetails?.display)
+                                    "
+                                >
+                                    <tr>
+                                        <td
+                                            class="border-none vertical-align-bottom"
+                                            *ngIf="fieldsAndVisibility?.header?.showEInvoiceDetails?.display"
+                                        >
+                                            <table>
+                                                <tr>
+                                                    <td class="border-none padding-0" width="90px">IRN</td>
+                                                    <td class="border-none padding-0" style="word-break: break-all">
+                                                        :
+                                                        def01b2b4b08ba07e92556896bb0e5a6c5b80428f756ae902ab57db0b60781c3
```

Comment:
please check conflict not resolved properly

Reply:
IRN number